modules/gdscript/gdscript_vm.cpp

```diff
@@ -1171,6 +1173,38 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 			}
 			DISPATCH_OPCODE;
 
+			OPCODE(OPCODE_SET_STATIC_VARIABLE) {
+				CHECK_SPACE(4);
+
+				GET_VARIANT_PTR(_class, 0);
+				GDScript *gdscript = Object::cast_to<GDScript>(_class->operator Object *());
+				GD_ERR_BREAK(!gdscript);
+
+				GET_VARIANT_PTR(index, 1);
+				GET_VARIANT_PTR(value, 2);
+
+				gdscript->static_variables.write[*index] = *value;
```

Comment:
With `index` being `int` instead of `Address`, this part won't need to create a Variant for the index:

```suggestion
				int index = _code_ptr[ip + 2];
				GD_ERR_BREAK(index < 0 || index >= gdscript->static_variables.size());
				GET_VARIANT_PTR(value, 2);

				gdscript->static_variables.write[index] = *value;
```

Might be better to put the index argument last so the Variant addresses are contiguous, which is common in other opcodes making it easier to understand the flow.

Same applies to the `GET` opcode below.